cordova-plugin-outline/android/java/org/outline/vpn/VpnTunnelService.java

```diff
@@ -362,6 +362,11 @@ public void onAvailable(Network network) {
       }
       broadcastVpnConnectivityChange(OutlinePlugin.ConnectionStatus.CONNECTED);
       displayPersistentNotification(null, OutlinePlugin.ConnectionStatus.CONNECTED);
+
+      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+        // Indicate that traffic will be sent over the current active network.
+        setUnderlyingNetworks(new Network[] {connectivityManager.getActiveNetwork()});
```

Comment:
I think you can just use `network` here, since you've already verified above that it is the active network.